### ISSUE TITLE
Feature Request: Simple Place to Locate "My Bids" and "My Asks"

### ISSUE THREAD

**Issue by jessicatoon1** [vessenes/persona#29]
This was mentioned several times by multiple Top Testers in the Discord channel. While you can more easily see Asks by looking at your Collection page, there isn't an easy place to see what Bids you have placed so you can track if you have been outbid, etc.

**Comment by vessenes** [comment]
Makes sense. I think a marketplace management interface is needed in general.
